app/models/service_template.rb

```diff
@@ -10,6 +10,20 @@ class ServiceTemplate < ApplicationRecord
     "storage"         => _("Storage")
   }.freeze
 
+
+  CATALOG_ITEM_TYPES = {
```

Comment:
@gmcculloug usually we try to keep list of types in a constant in models, but in this case it has been in controller from the beginning, but now since we need to expose the type as a nice display value in the list view. I thought we should move this into the model and reference the constant from controllers/views.